_episodes/02-filedir.md

```diff
@@ -27,122 +25,39 @@ keypoints:
 - "Directory names in a path are separated with `/` on Unix, but `\\\\` on Windows."
 - "`..` means 'the directory above the current one'; `.` on its own means 'the current directory'."
 - "Most files' names are `something.extension`. The extension isn't required, and doesn't guarantee anything, but is normally used to indicate the type of data in the file."
-- "Most commands take options (flags) which begin with a `-`."
 ---
 
-The part of the operating system responsible for managing files and directories
+The part of the operating system responsible for managing files and directories 
 is called the **file system**.
 It organizes our data into files,
 which hold information,
 and directories (also called "folders"),
 which hold files or other directories.
 
 Several commands are frequently used to create, inspect, rename, and delete files and directories.
-To start exploring them,
-let's open a shell window:
+To start exploring them, we'll go to our open shell window:
 
-> ## Preparation Magic
->
-> If you type the command:
-> `PS1='$ '`
-> into your shell, followed by pressing the 'enter' key,
-> your window should look like our example in this lesson.  
-> This isn't necessary to follow along (in fact, your prompt may have
-> other helpful information you want to know about).  This is up to you!  
-{: .callout}
```

Comment:
I think 'preparation magic' should be kept -- it's certainly useful for the instructor.

Reply:
oh, I had moved it into episode 1 (albeit into the normal text flow, without the callout), to sit with where prompts etc are discussed

Do you think I should put it into a callout there, or leave it in the text flow?

Reply:
Sorry - my bad. There's a good bit of rearrangement to keep track of :)
I think it's fine there. It *could* go in a callout as before, or just leave it as it is. Perhaps @shwina or @gdevenyi have thoughts on that?